### PR TITLE
Ap icall

### PR DESCRIPTION
**Pull request by folusho-adeyemi**
## Description

Frontend Files for third party API implementation

### CODE REVIEW

File: FashionConnect_ui/src/App.jsx [hunk 1]
```diff
@@ -33,62 +33,23 @@ function App() {
...
-      return null;
-    }
-  };
-
   return (
     <div className="app">
       <UserContext.Provider value={{ user, updateUser }}>
         <BrowserRouter>
           <Routes>
             {/* <Route path="/" element={ <Main /> } /> */}
```
Comment (mjyuen) [inline FashionConnect_ui/src/App.jsx:43]: You can remove this line

File: FashionConnect_ui/src/components/AddToCollectionsPopup/AddToCollectionsPopup.jsx [hunk 2]
```diff
@@ -1,96 +1,111 @@
...
+      setSelectedCollections((prevSelected) => [...prevSelected, collectionID]);
+    } else {
+      setSelectedCollections((prevSelected) =>
+        prevSelected.filter((id) => id !== collectionID)
+      );
+    }
+  };
+
+  const handleAddToCollection = () => {
     // To Update the collections with the new ProductID
```
Comment (mjyuen) [inline FashionConnect_ui/src/components/AddToCollectionsPopup/AddToCollectionsPopup.jsx:37]: when you have a comment that describes a function and what it's doing, it's best practice to put the comment before the function declaration

File: FashionConnect_ui/src/components/AddToCollectionsPopup/AddToCollectionsPopup.jsx [hunk 3]
```diff
@@ -1,96 +1,111 @@
...
+      } catch (error) {
+        throw error;
+
+      }
+    };
+
+    // To Call a function to save the updated collections to the database
+    saveUpdatedCollectionsToDatabase(updatedCollections);
+
+    // To Close the popup
```
Comment (mjyuen) [inline FashionConnect_ui/src/components/AddToCollectionsPopup/AddToCollectionsPopup.jsx:82]: i think this comment isn't needed, the code is self-explanatory

File: FashionConnect_ui/src/components/AddToCollectionsPopup/AddToCollectionsPopup.jsx [hunk 4]
```diff
@@ -1,96 +1,111 @@
...
-      );
-    }
-    
+      } catch (error) {
+        throw error;
+
+      }
+    };
+
+    // To Call a function to save the updated collections to the database
```
Comment (mjyuen) [inline FashionConnect_ui/src/components/AddToCollectionsPopup/AddToCollectionsPopup.jsx:79]: // Save the updated collections to the database

**Review by mjyuen — APPROVED** [review]
looks good overall, good to see refactoring and clean-up
I left some comments for just general guidance

File: FashionConnect_ui/src/components/AddToCollectionsPopup/AddToCollectionsPopup.jsx (hunk 2 above)
Comment (folusho-adeyemi) [reply FashionConnect_ui/src/components/AddToCollectionsPopup/AddToCollectionsPopup.jsx:37]: Alright thanks!

File: FashionConnect_ui/src/components/AddToCollectionsPopup/AddToCollectionsPopup.jsx (hunk 4 above)
Comment (folusho-adeyemi) [reply FashionConnect_ui/src/components/AddToCollectionsPopup/AddToCollectionsPopup.jsx:79]: Thanks!

**Pull request merged**